### PR TITLE
Fix condition on schedulable in oc adm manage-node

### PR DESCRIPTION
**Pull request by mfojtik**
Fixes: https://github.com/openshift/origin/issues/11070

With this fix applied, you get the right behavior:

```console
[@dev] .../openshift/origin # oc adm manage-node dev --schedulable=true --config=/var/lib/openshift/openshift.local.config/master/admin.kubeconfig
NAME      STATUS    AGE
dev       Ready     35m
[@dev] .../openshift/origin # oc adm manage-node dev --schedulable=false --config=/var/lib/openshift/openshift.local.config/master/admin.kubeconfig
NAME      STATUS                     AGE
dev       Ready,SchedulingDisabled   35m
[@dev] .../openshift/origin # oc adm manage-node dev --schedulable=true --config=/var/lib/openshift/openshift.local.config/master/admin.kubeconfig
NAME      STATUS    AGE
dev       Ready     35m
```

### CODE REVIEW

File: pkg/cmd/admin/node/schedulable.go [hunk 1]
```diff
@@ -20,10 +22,11 @@ func (s *SchedulableOptions) Run() error {
 	errList := []error{}
 	var printer kubectl.ResourcePrinter
 	for _, node := range nodes {
-		if node.Spec.Unschedulable == !s.Schedulable {
+		if node.Spec.Unschedulable == s.Schedulable {
 			node.Spec.Unschedulable = !s.Schedulable
 			node, err = s.Options.Kclient.Nodes().Update(node)
 			if err != nil {
+				fmt.Printf("err=%v\n", err)
```
Comment (deads2k) [inline pkg/cmd/admin/node/schedulable.go]: remove

**Comment by deads2k** [comment]
We have a fake node in `test-cmd` right?  Seems worthy of a test.

File: pkg/cmd/admin/node/schedulable.go (hunk 1 above)
Comment (mfojtik) [inline pkg/cmd/admin/node/schedulable.go]: hups, removed.

**Comment by deads2k** [comment]
lgtm otherwise.

**Comment by mfojtik** [comment]
@deads2k added [test]

File: pkg/cmd/admin/node/schedulable.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func (s *SchedulableOptions) Run() error {
 	errList := []error{}
 	var printer kubectl.ResourcePrinter
 	for _, node := range nodes {
-		if node.Spec.Unschedulable == !s.Schedulable {
+		if node.Spec.Unschedulable == s.Schedulable {
```
Comment (deads2k) [inline pkg/cmd/admin/node/schedulable.go]: Just to be sure I got it.  This is what we're shooting for right?

 | node.Spec.Unschedulable == true | node.Spec.Unschedulable == false |
------------ | ------------- | ----------
 s.Schedulable == true | change to false | do nothing | 
 s.Schedulable == false | do nothing | change to true |

**Comment by liggitt** [comment]
Dupe of https://github.com/openshift/origin/pull/10971, right?

**Comment by mfojtik** [comment]
@liggitt heh yes, I didn't noticed that PR :-) will close this in favor of @marun PR.

**Pull request closed without merging**